docker-compose.prod.yaml

```diff
@@ -0,0 +1,65 @@
+version: "3"
+services:
+  reverse-proxy:
+    image: traefik:v2.0
```

Comment:
Why you not use a newer image, this is a year old
https://hub.docker.com/layers/traefik/library/traefik/v2.0/images/sha256-384830d8a7692d3169768e284b49f7a96a73e74d8d3a9f4b143d9eac86f3b919?context=explore

Reply:
Done.